src/script/event/EventRepositorySpec.ts

```diff
@@ -160,25 +163,25 @@ describe('EventRepository', () => {
     });
 
     it('should not update last notification id if transient is true', () => {
-      const notification_payload = {id: createRandomUuid(), payload: [], transient: true};
+      const notification_payload = {id: createRandomUuid(), payload: [], transient: true} as Notification;
```

Comment:
Suggestion: A const with `TypeUtil.RecursivePartial<Notification>` from commons might be a thing here. When the notification is used later this can be casted to `Notification`. The benefit here would be a partial type check of the `Notification` attributes that are used for the test.